src/backend/app/projects/project_schemas.py

```diff
@@ -133,7 +133,7 @@ class ProjectIn(BaseModel):
     """Upload new project."""
 
     project_info: ProjectInfo
-    organisation_id: Optional[int] = None
+    organisation_id: int
```

Comment:
I removed this intentionally, as it doesn't actually do anything.
If you check the logic for create_project, the org id is taken from the url route param, not from the json body 👍

Reply:
I understand, but projects were created even without an organization ID. I don't know how, which is why it was made mandatory.

Reply:
We would have to specify org_id in both the url and the data json, potentially causing conflicts if different values are passed.

This line should add the org_id to the project: https://github.com/hotosm/fmtm/blob/bd6c74379dd167a787b1cab050eab4f65bc9cb92/src/backend/app/projects/project_routes.py#L267

When were you testing? Perhaps I fixed it since you last tested (I have been heavily refactoring the project creation, sorry if it causes issues 🙏 )

Reply:
The day before yesterday.

Reply:
Please test the create_project endpoint and report if errors 👍